Lib/codeop.py

```diff
@@ -86,24 +87,12 @@ def _maybe_compile(compiler, source, filename, symbol):
     with warnings.catch_warnings():
         warnings.simplefilter("error")
 
-        code1 = err1 = err2 = None
         try:
```

Comment:
Line 68 above uses an unexposed and undocumented flag used indirectly in _maybe_compile.
```
PyCF_DONT_IMPLY_DEDENT = 0x200 # Matches pythonrun.h.
```
Do you know what this does and whether it is actually needed, especially after these changes?

Reply:
Regarding if is actually needed, I don't know. I prefer not to remove it in this PR, at least

Reply:
Check out https://github.com/python/cpython/commit/4b499dd3fb49181efbd99f87e45ff923146cba8a

Reply:
Thank you for the link.  When I tested a single compile call instead of two, I may have left out that flag.

Reply:
Being added specifically for codeop explains why it was not exposed.